README.md

```diff
@@ -141,6 +141,11 @@ ckanext.security.redis.db = 1                 # ckan uses db 0
 ckanext.security.lock_timeout = 900           # Login throttling lock period
 ckanext.security.login_max_count = 10         # Login throttling attempt limit
 ckanext.security.brute_force_key = user_name  # Detect brute force attempts by username rather than IP address
+
+# If using 2.7.7 or recent patches of 2.8, the password reset behaviour has been fixed in CKAN core
+# (no longer discloses info about non-existent accounts)
+# You can disable the fix in this plugin by:
+ckanext.security.disable_password_reset_override = true
```

Comment:
Is there any benefit / difference in having this enabled/disabled in 2.7.7 or 2.8+ ? Is it suggested that this is disabled if you're on those newer versions?

Reply:
Yeah, possibly this could be worded better... the implementation in here actually breaks in the versions that have been fixed in CKAN core. I'll update the text to indicate that and merge this in, thanks Matt.